app/controllers/UnsecuredApplication.java

```diff
@@ -0,0 +1,58 @@
+package controllers;
+
+import play.Configuration;
+import play.Play;
+import play.db.jpa.Transactional;
+import play.mvc.Result;
+import play.mvc.Controller;
+
+
+public class UnsecuredApplication extends Controller {
+
+   	private static String INFO = null;
+
+    public static String info() {
+    	init();
+        return INFO;
+    }
+
+   	private static void init(){
+   		if (INFO == null){
+	   	    final Configuration cfg = Play.application().configuration();
+			String version = "Version: " + cfg.getString("app.version");
+	   	    String dbName = "Database: " + cfg.getString("db.default.url");
+	   	   	INFO = "Copyright 2014-2016 - University of Pittsburgh, " 
+	   	    + version + ", " + dbName;
+   		}
+   	}
+   	
+   	
+   	public Result about() {
+		return ok(views.html.about.render("about", info()));
+	}
```

Comment:
I don't think we need any methods below this line. They are just noise but do no harm. I think it's better to remove them and make sure that it still works. 

BTW,  I cannot accept your pull request. You should request to John.